### PR TITLE
relative time string passed through to variable from comparison filter

### PR DESCRIPTION
**Pull request by hjkmarshall**
Used inside KPI comparison chart

### CODE REVIEW

**Review by eugeneoshepkov — APPROVED** [review]

File: src/components/vanilla/charts/KPIChart/index.tsx [hunk 1]
```diff
@@ -31,7 +32,7 @@ export default (props: Props) => {
     const prev = parseFloat(prevResults?.data?.[0]?.[metric.name] || 0);
 
     return {
-      percentage: prev ? Math.round((n / prev) * 100) - 100 : null,
+      percentage: prev !== null && prev !== undefined ? Math.round((n / prev) * 100) - 100 : null,
```
Comment (mad-raccoon) [inline src/components/vanilla/charts/KPIChart/index.tsx:35]: prev !== null && prev !== undefined 


is equal to **prev** , is this because of the 0 ?

Comment (hjkmarshall) [reply src/components/vanilla/charts/KPIChart/index.tsx:35]: @mad-raccoon yes, I want to it calculate infinity for values of 0, as that it then shown below.

Comment (hjkmarshall) [reply src/components/vanilla/charts/KPIChart/index.tsx:35]: @mad-raccoon you ok for me to resolve and merge?

Comment (mad-raccoon) [reply src/components/vanilla/charts/KPIChart/index.tsx:35]: yes, sorry for the delay.  this has more readability - but i am ok with that as well:

```suggestion
      percentage: prev || prev === 0 ? Math.round((n / prev) * 100) - 100 : null,
```

**Pull request merged**